### DISCUSSION TITLE
Guidelines for developers

### DISCUSSION THREAD

**Discussion by leliocampanile** [lcfd/trak#9] [General]
Hello,
I am trying to test the application using the source code from GitHub, but there is not guideline or readme to developers.

I think that a read for developing purpose It would be a good idea.

**Comment by lcfd** [comment]
Hi Lelio! Sure, thank you for the feedback.

I'm in the process of deciding where to focus at the moment.

More documentation for developers **and** users must be on the list of TODOs.

Where would be the best place to put them?
Github wiki, .md pages in the repo in a docs folder, on the website.

Let me know your opinion! 🙏

**Reply by leliocampanile** [reply]
Hi Luca,
in my opinion the best would be write  .md file in docs, a section for developing.
Basically it needs a quickstart for developing.

**Reply by lcfd** [reply]
I have created an issue to add what you are asking for #12

It should be kind of a "How to contribute" section. Right?

Ok for the markdown format, it was already what I wanted to do.

I've already created a docs section on the website and put some information in the README but I think that's not a good solution.

My desire is to have a docs.usetrak.com website that uses the .md files we were talking about,  to declutter a bit the actual website that should have communication-only pages.

Thank you again for the suggestions. It's on schedule 👍
